go/database/mpt/io/live.go

```diff
@@ -302,9 +302,24 @@ func runImport(directory string, in io.Reader, config mpt.MptConfig) (root mpt.N
 type exportVisitor struct {
 	out io.Writer
 	err error
+	ctx context.Context
+}
+
+func (e *exportVisitor) interrupt() bool {
```

Comment:
why is not the already defined function `func interrupt(ctx context.Context) bool` re-used?

Reply:
Good catch.